openfisca_uk/parameters/hmrc/tax_credits/working_tax_credit/elements/basic.yaml

```diff
@@ -0,0 +1,29 @@
+description: The basic element of Working Tax Credit.
+values:
+  2002-08-01: 1_525
+  2004-04-06: 1_570
+  2005-04-06: 1_620
+  2006-04-06: 1_665
+  2007-04-06: 1_730
+  2009-04-06: 1_890
+  2010-04-06: 1_920
+  2011-04-06: 1_920
+  2012-04-06: 1_920
+  2013-04-06: 1_920
+  2014-04-06: 1_940
+  2015-04-06: 1_960
+  2016-03-16: 1_960
+  2017-04-06: 1_960
+  2018-04-06: 1_960
+  2019-04-06: 1_960
+  2020-03-25: 1_960
+  2020-04-06: 3_040 # Coronavirus Act 2020, overrides 1_960 in WTC Regulations.
```

Comment:
was there a reference for the Coronavirus Act 2020?

Reply:
Added